.github/workflows/precommit.yml

```diff
@@ -6,11 +6,11 @@ jobs:
   lint:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - name: Setup Python
         uses: actions/setup-python@v4
         with:
-          python-version: 3.8
+          python-version: "3.12"
```

Comment:
docs.yml uses 3.11. This uses 3.12. Should both be the same?

Reply:
One of the dependencies of docs does not support 3.12 yet.